ldapauthenticator/ldapauthenticator.py

```diff
@@ -128,11 +127,19 @@ def authenticate(self, handler, data):
                     userdn = conn.response[0]['dn']
 
                 for group in self.allowed_groups:
+                    groupfilter = (
+                        '(|'
+                        '(member={userdn})'
+                        '(uniqueMember={userdn})'
+                        '(memberUid={uid})'
+                        ')'
+                    ).format(userdn=userdn, uid=username)
```

Comment:
I am passing the uid/username. uid in the sense of rfc2307. According to the RFC:

> To avoid confusion, the term "login name" refers to the user's login
   name (being the value of the uid attribute) and the term "user ID"
   refers to he user's integer identification number (being the value of
   the uidNumber attribute).

So in this sense, group members are listed in the group with the following ldif:
> dn: cn=some_group,ou=Groups,dc=example,dc=com
   cn: some_group
   gidNumber: 10000
   objectClass: posixGroup
   objectClass: top
   memberUid: someusername
   memberUid: someotherusername